services/QuillLMS/app/assets/stylesheets/pages/privacy_and_security.scss

```diff
@@ -0,0 +1,261 @@
+      align-items: center;
+      img {
+        height: 88px;
+        width: 88px;
+        @include quill-green-filter;
+      }
+      .header {
+        color: $quill-green;
+        text-align: center;
+        font-family: "Adelle Sans";
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 130%;
+        margin: 32px 0px;
+      }
+      .subtext {
+        color: $quill-grey-50;
+        text-align: center;
+        font-family: "Adelle Sans";
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 150%;
+      }
+    }
+  }
+  .details-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 96px 0px;
+    img {
+      height: 88px;
+      width: 88px;
+    }
+    h2 {
+      color: $quill-green;
+      text-align: center;
+      font-family: "Adelle Sans";
+      font-size: 40px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 150%;
+      margin: 32px 0px;
+    }
+    .subtext {
+      color: $quill-grey-50;
+      text-align: center;
+      font-family: "Adelle Sans";
+      font-size: 20px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 150%;
+      &.narrow {
+        max-width: 850px;
+      }
+    }
+    .details-row {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+      justify-content: space-between;
+      margin: 64px 0px;
+    }
+    .details-column {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      .detail-container {
+        margin: 64px 0px 0px 0px;
+        max-width: 850px;
+        h3, p {
+          text-align: center;
+        }
+      }
+    }
+    .detail-container {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      max-width: 584px;
+      &:first-of-type {
+        margin-right: 32px;
+      }
+      img {
+        height: 64px;
+        width: 64px;
+      }
+      h3 {
+        margin: 16px 0px;
+        color: $quill-grey-90;
+        font-family: "Adelle Sans";
+        font-size: 24px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 150%;
+      }
+      ul {
+        padding-left: 24px;
+        li {
+          color: $quill-grey-50;
+          font-family: "Adelle Sans";
+          font-size: 20px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: 150%;
+        }
+      }
+    }
+    .buttons-container {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      gap: 16px;
+    }
+  }
+  @media (max-width: 1200px) {
+    padding: 0px 64px;
+  }
+  @media (max-width: 980px) {
+    .icons-and-descriptions-container {
+      flex-direction: column;
+      gap: 64px;
+    }
+  }
+  @media (max-width: 700px) {
+    .rating-and-seal-container {
+      flex-direction: column !important;
```

Comment:
It's better practice to increase selector specificity than to use !important, esp. in a new file.